src/isar/state_machine/states/monitor.py

```diff
@@ -91,14 +89,13 @@ def _run(self):
         self.state_machine.to_next_state(next_state)
 
     def _mission_finished(
-        self, mission_status: MissionStatus, instance_id: int
+        self,
+        mission_status: MissionStatus,
     ) -> bool:
         if mission_status == MissionStatus.Unexpected:
-            self.logger.error(
-                f"Mission status on task {instance_id} returned and unexpected status string"
-            )
+            self.logger.error("Mission status returned an unexpected status string")
         elif mission_status == MissionStatus.Failed:
-            self.logger.warning(f"Mission instance {instance_id} failed...")
+            self.logger.warning("Mission failed...")
```

Comment:
Strictly speaking the Mission didn't fail, just a Task within the mission so this log message is not correct.

Would recommend logging something like "Task failed. Task ID: id" or something along those lines.

Reply:
We discussed renaming `MissionStatus` to `TaskStatus` as it's the status of a single `Task` and not the mission as a whole. If you wish to include that here you may, otherwise I can create a new issue to do so.

Reply:
In general, the term `mission` is not used consistently and does not have a unique meaning in ISAR. Renaming is in process and will be added to this PR. `MissionStatus ` -> `TaskStatus` is among those changes.